plugins/convertColors.js

```diff
@@ -39,6 +43,12 @@ const convertRgbToHex = ([r, g, b]) => {
   return '#' + hexNumber.toString(16).slice(1).toUpperCase();
 };
 
+/**
+ * @type {(node: XastNode) => boolean}
+ */
+const isMask = (node) =>
+  node && node.type === 'element' && node.name === 'mask';
```

Comment:
```suggestion
 * @type {(node: XastElement) => boolean}
 */
const isMask = (node) => node.name === 'mask'
```

(Also requires changing the `@typedef` at the top if this is used!)

In both places that you call `#isMask`, you actually have a `XastElement`, a more concrete type of `XastNode` that allows you to safely make some assumptions.

In both cases, you can safely check `node.name` without any defensive checks.

(Though it might be better to just inline the condition now that it's so short.)